src/applications/personalization/profile360/sass/profile-360.scss

```diff
@@ -39,7 +39,7 @@ input.usa-only-phone {
   line-height: 1;
   color: $color-black;
   text-align: center;
-  background-color: #eee;
+  background-color: $color-gray-lightest;
```

Comment:
close enough https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/52f75afbcd5e4217be513729e50a0fcc0dc92cf2/packages/formation/sass/base/_b-variables.scss#L71